test/main/dump.js

```diff
@@ -1,3 +1,5 @@
+/* globals Symbol:false */
+
```

Comment:
I believe we could set `esversion: 6` option on the JSHint config file. This inline setting wouldn't be necessary.